core/domain/feedback_services.py

```diff
@@ -302,6 +302,48 @@ def get_message(thread_id, message_id):
         feedback_models.GeneralFeedbackMessageModel.get(thread_id, message_id))
 
 
+# TODO(anubhavsinha98): Remove this function once the message field in all
+# the threads are populated.
+def get_message_count(thread_id):
+    """Fetches the messages count of the given thread.
+
+    Args:
+        thread_id: str. The ID of the thread to count messages in.
+
+    Returns:
+        int. The count of messages of the given thread.
+    """
+    return feedback_models.GeneralFeedbackMessageModel.get_message_count(
+        thread_id)
+
+
+# TODO(anubhavsinha98): Remove this function once the message field in all
+# the threads are populated.
+def thread_message_count(thread_id):
+    """Returns the messages count of the given thread.
+
+    Args:
+        thread_id: str. The ID of the thread to count messages in.
+
+    Returns:
+        int. The count of messages of the given thread.
+    """
+    thread = feedback_models.GeneralFeedbackThreadModel.get(thread_id)
+    return thread.message_count
+
+
+# TODO(anubhavsinha98): Remove this function once the message field in all
+# the threads are populated.
+def set_message_count(thread_id, message_count):
```

Comment:
Same issue. Don't make this a public function that anyone can call. It should not be exposed in the codebase.

Reply:
I will add these functions in the test class.

Reply:
DONE